src/config/settings.ts

```diff
@@ -18,7 +18,7 @@ const console = Logging.settings;
 // it will be picked up from the default settings object.
 // Version incrementing is for when a breaking change is introduced in the settings object.
 
-const CURRENT_SETTINGS_VERSION = 3;
+const CURRENT_SETTINGS_VERSION = 4;
```

Comment:
Not sure we need the version bump (that is, I don't think we need an explicit migration step — didn't add one to `updateTable` either).

Reply:
:+1: implemented

Reply:
Normally, we only need version bumps for settings when we introduce a breaking change in the settings structure. Don't see any in this PR.